### PR TITLE
[WIP] Tide-up helm chart

### PR DESCRIPTION
**Pull request by mjudeikis**
Currently Helm chart is not working with community version. There are a lot of values missing and virtual workspaces do no work outside process.

This PR is not working too as yet. Its raised more as visibility that I'm working on refactoring it a bit so if somebody wants to pick-up/reuse/help - feel free. Once I get to the place where it deploys confidently and I will clien commits and prepare to merge

For now its blocked by:
https://github.com/kcp-dev/kcp/issues/2254 
https://github.com/kcp-dev/kcp/issues/2295 
https://github.com/kcp-dev/kcp/issues/2325
https://github.com/kcp-dev/kcp/issues/2184

TODO:

- [ ] Make use of community images
- [ ] make VW running in separate process
- [ ] Make Shards scalable (Deployment to scaleSet? ) 
- [ ] Generate development/test kubeconfig for auth-proxy

### CODE REVIEW

File: charts/kcp/values.yaml [hunk 1]
```diff
@@ -25,35 +29,53 @@ kcp:
...
     port: 6060
+  tokenAuth:
+    enabled: false
+    fileName: auth-token.csv
+    config: |
+        user-1-token,user-1,1111-1111-1111-1111,"team-1"
+        admin-token,admin,5555-5555-5555-5555,"system:kcp:admin,system:kcp:clusterworkspace:admin"
 kcpFrontProxy:
-  image: registry.ci.openshift.org/kcp/kcp-front-proxy
+  image: ghcr.io/faroshq/kcp:latest
```
Comment (mjudeikis) [inline charts/kcp/values.yaml:39]: dev image

**Comment by mjudeikis** [comment]
Char is pointing to main branch images and is not up-to-date after refactoring changes merged recently. Need more work to adjust to new workspace structure :/

**Comment by yogesh-reddy** [comment]
@mjudeikis so i was working on deploying the kcp workload on k8s cluster .. is this helm chart the only way or do we have some other ways to achieve that ?

**Comment by mjudeikis** [comment]
There multiple aspects to this. Let me respond In more details tomorrow.  Not at pc now :)

**Comment by yogesh-reddy** [comment]
@mjudeikis I may not need production quality with sharding now .. I can just live with one instance.
my use case is as simple as running minimal kcp like how we run with binaries locally.
so is this possible .. if not by helm charts at least by some bunch of deployment YAML?

**Comment by mjudeikis** [comment]
I dont have those. And as to my knowledge there is nothing public yet. As said for time being https://github.com/kcp-dev/kcp/tree/main/cmd/sharded-test-server is best referece how to do so.

**Comment by yogesh-reddy** [comment]
@mjudeikis i have few questions regd the connectivity to kcp which in running on some cluster through this chart...

1. what endpoint i need mention (provided i create ingress on eks flavoured cluster)
2. How do i access the kcp server ? i mean where do i get kubeconfig etc or if it is through proxy server how do i do it ?

**Pull request closed without merging**